### DISCUSSION TITLE
Performant excel-like table for web

### DISCUSSION THREAD

**Discussion by mispp** [DioxusLabs/dioxus#2177] [Q&A]
Hi all,

i am evaluating frontend frameworks for one web app i have in mind and this popped up.

Given the current state of the toolikit, is it possible to make something like https://tanstack.com/virtual/latest or https://grid.glideapps.com/ using dioxus? Not all features are needed though.
Functionality i have in mind:
- scrollable via keyboard (like in excel)
- programmatically scrolling (e.g. telling the table to jump to row 345345, or go to col named "AMOUNT")
- fast rendering of many rows (lets say 1M rows and 200 cols)
- custom styling (freezing header and first col) -> im assuming this is possible with tailwind
- auto-filtering

Glide grid and tanstack virtual seem to use html canvas, so wondering how this would be used in dioxus.

Thanks

**Comment by ealmloff** [comment]
The https://github.com/matthunz/dioxus-lazy library provides virtual list components for Dioxus. I think it is using Dioxus 0.4, so you may need to upgrade it to Dioxus 0.5 first
